clients/apple/macOS/TreeView/FileTreeView.swift

```diff
@@ -65,6 +75,23 @@ struct FileTreeView: NSViewRepresentable {
             }
         }
     }
+    
+    func expandAncestorsOfDocument(file: File) {
+        if(treeView.row(forItem: file) != -1) {
```

Comment:
If I'm reading this correctly, if a file is presently invisible we wouldn't expand it's parent?

Reply:
This `if` statement will execute if a file visible on the tree. That being, that all its parents are expanded. In that case, we don't need to expand any of its parents, and only need to scroll to it if it is presently not visible.